pkg/shares/compact_shares_test.go

```diff
@@ -62,7 +62,7 @@ func TestFuzz_processCompactShares(t *testing.T) {
 
 func Test_processCompactShares(t *testing.T) {
 	// exactTxShareSize is the length of tx that will fit exactly into a single
-	// share, accounting for namespace id and the length delimiter prepended to
+	// share, accounting for the tx length delimiter prepended to
```

Comment:
[question]
does it no longer account for the namespace as well?

Reply:
this comment rename could've been extracted to a separate PR but the reason I removed namespace id from the comment is because `ContinuationCompactShareContentSize` already accounts for subtracting the namespace bytes from the share size.

https://github.com/celestiaorg/celestia-app/blob/395e26819f87d8521e6ef587bbb33905f5e5da08/pkg/appconsts/appconsts.go#L35-L39

The `- 1` is an attempt at accounting for the tx length delimiter prepended to each tx.